test/Spring-test.js

```diff
@@ -167,26 +258,64 @@ describe('Spring', () => {
       },
     });
     TestUtils.renderIntoDocument(<App />);
+
+    // Initial render
+    expect(count).toEqual([0]);
+    mockRaf.manySteps(8);
+    expect(count).toEqual([
+      0,
```

Comment:
Ok I'll bring them down to 5 (just to have 4 after the initial 0).

I am concerned by precision too. _Can_ it really vary though? Aren't those things deterministically wrong? I'd say truncating if anything.